src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs

```diff
@@ -756,13 +764,11 @@ private WorkItemRelationType ParseLink(WiLink link)
             return linkType;
         }
 
-        private WorkItemRelation IdentifyAttachment(WiAttachment att, WorkItem wi, IsAttachmentMigratedDelegate<string, string, bool> isAttachmentMigratedDelegate)
+        private WorkItemRelation IdentifyAttachment(WiAttachment att, WorkItem wi)
         {
-            if (isAttachmentMigratedDelegate(att.AttOriginId, out string attWiId))
```

Comment:
A WorkItemRelation that has WorkItemRelation.Rel = "AttachedFile" and also has a "name" attribute represents a workitem attachment that has been returned by Azure DevOps. That also means it has been migrated (otherwise it would not exist as attachment on the workitem).

Reply:
Is this still relevant? This fix will cut out the use of the attachment journal that is being updated during the import. If that is not relevant to use anymore we should clean that up too.
The question remains is what happens if you want to stop in the middle of import and continue afterward, as that was the purpose of the journal for work items and attachments?

Reply:
Let's axe this, I'm happy with #592 for now